endpoint-sec-sys/src/message.rs

```diff
@@ -3100,7 +3100,6 @@ pub struct es_message_t {
     /// See `global_seq_num`.
     ///
     /// Field available only if message version >= 2.
-    #[cfg(feature = "macos_10_15_1")]
     pub seq_num: u64,
```

Comment:
on macOS 10.15.0 this is 'reserved', so we should add a private u64 instead no ? Or at least document that it's value is not guaranteed to make sense, even if it's a valid u64